packages/compass-aggregations/src/modules/side-panel.ts

```diff
@@ -1,5 +1,9 @@
 import type { AnyAction } from 'redux';
 import { isAction } from '../utils/is-action';
+import { type PipelineBuilderThunkAction } from '.';
```

Comment:
Just a question, does this work the same way as:
```
import type { PipelineBuilderThunkAction } from '.'
```

Reply:
In this case, yes. But with `import type {} from ''`, you can only import types and with `import {type A, B} from ''`, you can import both.